haskell/src/Stablecoin/Util.hs

```diff
@@ -0,0 +1,51 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: MIT
+
+module Stablecoin.Util
+  ( readContract'
+  , aesonOptions
+  ) where
+
+import Data.Aeson
+import Data.Aeson.Casing
+import Data.Singletons (demote)
+import Fmt (pretty)
+
+import Michelson.Runtime
+import Michelson.TypeCheck (SomeContract(SomeContract), typeCheckContract)
+import Michelson.Typed (KnownT)
+import qualified Michelson.Typed as T
+import Util.Default (Default(def))
+import Util.Typeable ((:~:)(Refl), eqT)
+
+aesonOptions :: Options
+aesonOptions = (aesonPrefix camelCase) { omitNothingFields = True}
+
+-- | Similar to 'Michelson.Test.Import.readContract',
+-- but instead of showing just the expected type @cp@ when an error occurs,
+-- it shows both the expected @cp@ and unexpected @cp'@ types.
```

Comment:
I see the following code snippet which AFAIU is supposed to be print both types:
```
instance Buildable ImportContractError where
  build =
    \case
      ICEUnexpectedParamType actual expected ->
        "Unexpected parameter type: " +| actual |+
        ", expected: " +| expected |+ ""
```

Does it work incorrectly there? Or am I reading it wrong?

Reply:
Ah you're right! I completely missed that because it's all in one :so-big: line 😅

I'll delete `readContract'` from stablecoin, and make an MR in morley to spread the error message across multiple lines to make it easier to read.

![image](https://user-images.githubusercontent.com/718166/100326057-5c613480-2fc1-11eb-9f38-656fe8cc78af.png)